scg-system/src/main/java/io/telicent/backup/services/DatasetBackupService.java

```diff
@@ -701,7 +741,7 @@ private DataAccessPoint getDataAccessPoint(String datasetName) {
      */
     private void compressAndStoreBackupMetadata(ObjectNode response, String dirPath) {
         final Path zipFilePath = zipDirectory(dirPath, dirPath + ZIP_SUFFIX, DELETE_GENERATED_FILES);
-        writeObjectNodeToFile(response, dirPath + JSON_INFO_SUFFIX);
+        //writeObjectNodeToFile(response, dirPath + JSON_INFO_SUFFIX);
```

Comment:
If this now happens at the end of the method just remove the commented out line to avoid any confusion